horreum-backend/src/test/java/io/hyperfoil/tools/horreum/server/RouteFilterTest.java

```diff
@@ -0,0 +1,22 @@
+package io.hyperfoil.tools.horreum.server;
+
+import io.hyperfoil.tools.horreum.svc.BaseServiceTest;
+import io.hyperfoil.tools.horreum.test.NoGrafanaProfile;
+import io.hyperfoil.tools.horreum.test.PostgresResource;
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+import io.quarkus.test.oidc.server.OidcWiremockTestResource;
+
+@QuarkusTest
+@QuarkusTestResource(PostgresResource.class)
+@QuarkusTestResource(OidcWiremockTestResource.class)
+@TestProfile(NoGrafanaProfile.class)
+public class RouteFilterTest extends BaseServiceTest {
+
+   @org.junit.jupiter.api.Test
+   public void testRouteRedirectsNotFoundPageToHome() {
+      bareRequest().get("/duff-beer").then()
+         .statusCode(200);
```

Comment:
running this test locally the assertion passes.

Reply:
DYK this test is failing in CI env? I would be cautious about disabling tests in CI as we are then relying on contributors running the testsuite locally and we don't have full coverage in CI.

Reply:
Yes it was failing. Along with tests in UserTeamsFilterTest.
 Checking the logs produced by CI today shows test case is now skipped.

Reply:
having configured the connection information these are now working.